src/main/webapp/user-page.html

```diff
@@ -16,38 +16,79 @@
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
   </head>
   <body onload="buildUI();fetchBlobstoreUrlAndShowForm();">
     <div id="content">
 			<nav id="navigation">
       	<ul>
         	<li><a href="/">Home</a></li>
 					<li><a href="/feed.html">Message Feed</a></li>
-        	<li><a href="/aboutus.html">About Our Team</a></li>	
+        	<li><a href="/aboutus.html">About Our Team</a></li>
 				</ul>
     	</nav>
-			
+
 			<div class="padded rounded panel centered">
     		<h1 id="page-title">User Page</h1>
     	</div>
 
-			<div id="about-me-container" class="centered">Loading...</div>
-      <div id="about-me-form" class="hidden">
-      	<form action="/about" method="POST">
-        	<textarea name="about-me" placeholder="about me" class="rounded" rows=4 required></textarea>
-        	<br/>
-        	<input type="submit" value="Submit" class="rounded">
-      	</form>
-			</div>
+      <br>
+
+      <div class="about-me">
+        <h2 style="inline-block">About me</h2>
+        <button onclick="on()"><i class="fa fa-edit"></i></button>
+
+        <div id="overlay">
+          <form id="form" action="/about" method="POST">
+            <div class="header">
+              <p> Edit your profile </p>
+              <div class="close" onclick="off()"><i class="fa fa-times"></i></div>
+            </div>
+            <div class="form-group">
+              <input class="form-control" name="firstname" placeholder="First Name" required>
+            </div>
+
+            <div class="form-group">
+              <input class="form-control" name="lastname" placeholder="Last Name" required>
+            </div>
+
+            <div class="form-group">
+              <input class="form-control" name="country" placeholder="Country" required>
+            </div>
+
+            <div class="form-group">
+              <input class="form-control" name="city" placeholder="City" required>
+            </div>
+
+            <div class="form-group">
+              <textarea name="about-me" placeholder="About me" class="form-control" rows=3 required></textarea>
+            </div>
+
+            <div class="form-group">
+              <textarea class="form-control" name="recommendation" placeholder="Best place to visit in your city?" rows=3 required></textarea>
+            </div>
+
+            <input type="submit" value="Submit" class="rounded" style="float:right; margin-right:20px; display: inline-block">
```

Comment:
Nit: The style should be made a CSS class

Reply:
Done!